episodes/01-introduction.md

```diff
@@ -1,101 +1,94 @@
 ---
-title: "Model Coupling"
-teaching: 10
-exercises: 2
+title: "Introduction to model coupling"
+teaching: 30
+exercises: 20
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using Markdown and `{sandpaper}`?
+- What is meant by a "model"?
+- What is the difference between a mathematical model and a submodel?
+- What is model coupling?
+- Why would we want to couple simulation models?
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain how to use markdown with The Carpentries Workbench
-- Demonstrate how to include pieces of code, figures, and nested challenge blocks
+- Explain the benefits of coupling simulation models
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ## Introduction
+Computer simulations are widely used in modern science to understand and predict the behaviour of natural systems. The computer program that simulates the system is referred to as a "model" of that system. Some models may be very simple e.g. describing the evolution of a 1 dimensional system. Others may be highly complex, involving many different physical processes. A well known use of computer simulation is in the field of weather forecasting, in which a very complex model composed of a multitude of different physical processes must be simulated by a computer program, generally requiring powerful supercomputers. The computer program that simulates a given physical system is known as a _model_ of that system. A given climate simulation code is a _model_ of the Earth's climate, for example.
 
-This is a lesson created via The Carpentries Workbench. It is written in
-[Pandoc-flavored Markdown](https://pandoc.org/MANUAL.txt) for static files and
-[R Markdown][r-markdown] for dynamic files that can render code into output. 
-Please refer to the [Introduction to The Carpentries 
-Workbench](https://carpentries.github.io/sandpaper-docs/) for full documentation.
+## Mathematical model vs computational model
+For this course, it is important to distinguish between a _mathematical_ model, and a _computational_ model.
+A mathematical model refers to the set of mathematical equations that describe the evolution of a given physical system.
+The computational model refers to the actual computer program that describes the evolution of a physical system.
+These are closely connected concepts - after all, many computational models are just solving a given mathematical model numerically. However, the distinction is important. Some computational models can be simulating many different aspects of a physical system, which could involve solving many different mathematical models.
```

Comment:
Maybe mention than "verification" is about comparing the two and making sure the implementation is correct?

Reply:
Good point. I've added this to the end of the paragraph.